util/webhook/webhook_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/sirupsen/logrus/hooks/test"
```

Comment:
oh - I didn't know about this - neat

Reply:
wish I'd know weeks ago!

Reply:
I have been lucky to find that one. The unit test now checks the handler doesn't crash and checks the Handler definitely process the event as expected. :)